dev/drivers/scripts/plots/global_ens/jevs_global_ens_atmos_naefs_grid2obs_past31days_plots.sh

```diff
@@ -4,7 +4,7 @@
 #PBS -q dev
 #PBS -A VERF-DEV
 #PBS -l walltime=00:15:00
-#PBS -l place=vscatter,select=1:ncpus=37:mem=25GB
+#PBS -l place=vscatter,select=1:ncpus=37:mem=50GB
```

Comment:
This says 37, but `scripts/plots/global_ens/exevs_global_ens_atmos_naefs_grid2obs_plots.sh` shows 32.

Reply:
Oops, should have been changed to 32.  Let me correct.

Reply:
Done.